diff -uNr libunicode-0.7.old/Makefile.am libunicode-0.7/Makefile.am
--- libunicode-0.7.old/Makefile.am	Mon May  1 22:04:19 2000
+++ libunicode-0.7/Makefile.am	Thu Jul 18 10:51:39 2002
@@ -1,6 +1,7 @@
 SUBDIRS = include lib 
 
-EXTRA_DIST = copyright.w
+EXTRA_DIST = copyright.w unicodeConf.sh.in
+
 
 # cweb target for running ctangle/cweave
 
diff -uNr libunicode-0.7.old/configure.in libunicode-0.7/configure.in
--- libunicode-0.7.old/configure.in	Mon Jun  5 06:16:02 2000
+++ libunicode-0.7/configure.in	Thu Jul 18 10:53:14 2002
@@ -18,6 +18,16 @@
 AC_ISC_POSIX
 AM_PROG_LIBTOOL
 
+UNICODE_LIBDIR='-L${libdir}'
+UNICODE_INCLUDEDIR='-I${includedir} ${UNICODE_DEFS}'
+UNICODE_LIBS='-lunicode'
+
+AC_SUBST(UNICODE_LIBDIR)
+AC_SUBST(UNICODE_INCLUDEDIR)
+AC_SUBST(UNICODE_LIBS)
+
+
+
 dnl Add some compiler options--changing the name of the package.
 AC_ARG_PROGRAM
 
@@ -32,6 +42,4 @@
 # wide character support
 AC_CHECK_FUNCS(towlower)
 
-AC_OUTPUT(include/Makefile lib/Makefile Makefile)
-
-
+AC_OUTPUT(include/Makefile lib/Makefile Makefile unicodeConf.sh)
diff -uNr libunicode-0.7.old/unicodeConf.sh.in libunicode-0.7/unicodeConf.sh.in
--- libunicode-0.7.old/unicodeConf.sh.in	Thu Jan  1 01:00:00 1970
+++ libunicode-0.7/unicodeConf.sh.in	Thu Jul 18 10:47:34 2002
@@ -0,0 +1,8 @@
+#
+# Configuration file for using the UNICODE library in GNOME applications
+#
+UNICODE_LIBDIR="@UNICODE_LIBDIR@"
+UNICODE_LIBS="@UNICODE_LIBS@"
+UNICODE_INCLUDEDIR="@UNICODE_INCLUDEDIR@"
+MODULE_VERSION="unicode-@VERSION@"
+
